src/gatsby/src/components/park/parkDates.js

```diff
@@ -260,7 +260,7 @@ export default function ParkDates({ data }) {
                   />
                   {parkStatusText}
                 </h4>
-                {parkDates && (
+                {(parkDates && parkOperation.hasParkGate) && (
```

Comment:
hasParkGate is a nullable boolean.  What is the intended behaviour when it's null?

Reply:
Good question, I'll ask Amanda.

Reply:
At the standup on January 15th, BC Parks decided to treat null as true.